components/squad/squad_row.lua

```diff
@@ -0,0 +1,118 @@
+local Class = require('Module:Class')
+local String = require('Module:String')
+local Player = require('Module:Player')
+local ReferenceCleaner = require('Module:ReferenceCleaner')
+
+local _ICON_CAPTAIN = '[[image:Captain Icon.png|18px|baseline|Captain|link=' ..
+	'https://liquipedia.net/rocketleague/Category:Captains|alt=Captain]]'
+local _ICON_SUBSTITUTE = '[[image:Substitution.svg|18px|baseline|Sub|link=|alt=Substitution]]'
+
+local _COLOR_BACKGROUND_COACH = '#e5e5e5'
```

Comment:
should make that a css class instead (to avoid hard-coded colors)

Reply:
Follow-up